sdk/spring/spring-test-template.yml

```diff
@@ -147,7 +148,9 @@ stages:
             inputs:
               mavenPomFile: pom.xml
               goals: verify
-              options: '-Dmaven.wagon.http.pool=false $(DefaultOptions) -Dmaven.javadoc.skip=true -Drevapi.skip=true -DskipSpringITs=false -pl $(ProjectList)'
+              options: '$(DefaultTestOptions) -DskipSpringITs=false'
```

Comment:
Another case where adding an additional options would simplify this configuration

Reply:
Exactly we should definitely look at a clean-up for this.